.github/workflows/syntax_check_cloud.yml

```
@@ -31,3 +31,17 @@ jobs:
            echo "The above syntax errors were found, only use parenthesis in class definition if there is inheritance (code-style)."
            exit 1  
         fi
+
+    - name: Check test cases are runnable
+      if: ${{ !cancelled() }}
+      run: | 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)